automation/taskcluster/lib/tasks.py

```diff
@@ -442,6 +454,70 @@ def craft_upload_apk_nimbledroid_task(self, assemble_task_id):
             dependencies=[assemble_task_id],
         )
 
+    def craft_raptor_task(self, signing_task_id, mozharness_task_id, variant):
+        build_location = '{}/{}/artifacts/{}'.format(
```

Comment:
Nit: the term "build" is a little ambiguous here: it _could_ mean the task, rather than the artifact instead.
Can we call this `apk_location` instead?

Reply:
Good thinking. Done.